localstripe/resources.py

```diff
@@ -1304,11 +1304,24 @@ def _api_void_invoice(cls, id):
 
         return obj
 
+    @classmethod
+    def _api_list_lines(cls, id, limit=None, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        obj = cls._api_retrieve(id)
+
+        lines = List('/v1/invoices/' + id + '/lines', limit=limit)
+        lines._list = obj.lines._list
```

Comment:
I'd be something simpler, like in
https://github.com/adrienverge/localstripe/blob/c6821928953c50058327f04ef11b9e70bc8e609b/localstripe/resources.py#L749-L753
or
https://github.com/adrienverge/localstripe/blob/c6821928953c50058327f04ef11b9e70bc8e609b/localstripe/resources.py#L741-L746

Is it possible, or is it a hidden reason for the need to create a new `List`?

Reply:
It was mainly to make use of the `List` class validation code for the `limit` parameter instead of copying it.

Reply:
Ah, OK! It's a good reason.